### ISSUE TITLE
Search tool not much to be found

### ISSUE THREAD

**Issue by splouf** [sessions-marteau/sessions-marteau-2.github.io#26]
I looked into the search too on the site and it seems that there is not much appearing. If I type podcast: no results found if I type Sam Shalabi: no results found but Shalabi works. Anyways it is not a big deal but it can be usefull tool if it can find anything on the site not just certain tags.

**Comment by vigliensoni** [comment]
The current functionality of the searching tool searches on the tags assigned to posts.

@splouf

**Comment by splouf** [comment]
ok maybe we can add a full search fonctionnality if let's say search for and artist it can be rerouted to the musique page etc... In the meanwhile we could add a podcast tag to every session so if people want to listen they will be redirected to the podcast page.

**Comment by vigliensoni** [comment]
If we add artist tags for every single listening session, people will be able to search by artist name.

The podcast page is just an HTML page, and so there are no tags. Anyway, having a Podcast link in the navbar should enable people to easily find the podcast, don't you think?

**Comment by splouf** [comment]
Yes but while they browse quickly if it is handy they can click on a podcast link and find the podcast related to it quickly. Like in the first lines of the session.

**Comment by vigliensoni** [comment]
We should write something, then.

**Comment by splouf** [comment]
I added a link to the podcast page next to the programme into in each session
